src/main/java/org/betonquest/betonquest/compatibility/redischat/RedisChatInterceptor.java

```diff
@@ -0,0 +1,56 @@
+package org.betonquest.betonquest.compatibility.redischat;
+
+import dev.unnm3d.redischat.api.RedisChatAPI;
+import net.md_5.bungee.api.chat.BaseComponent;
+import net.md_5.bungee.api.chat.TextComponent;
+import org.betonquest.betonquest.api.profiles.OnlineProfile;
+import org.betonquest.betonquest.conversation.Conversation;
+import org.betonquest.betonquest.conversation.Interceptor;
+import org.bukkit.entity.Player;
+
+public class RedisChatInterceptor implements Interceptor {
+
+    protected final Conversation conv;
+
+    protected final Player player;
+
+    /**
+     * RedisChatAPI instance
+     */
+    private static final RedisChatAPI api = RedisChatAPI.getAPI();
```

Comment:
I would suggest to not store the api instance statically. Instead do the `RedisChatAPI.getAPI()` call in the constructor along with the remaining assignments and store the instance in a normal (non-static) variable.

That way you should also be able to prevent it from ever being null (or explicitly do a null check in the constructor if it still is a valid concern).